pytorch_lightning/trainer/training_loop.py

```diff
@@ -201,6 +200,8 @@ def check_checkpoint_callback(self, should_check_val, force_save=False):
         should_activate = not is_overridden('validation_step', model) and not should_check_val
         if should_activate or force_save:
             checkpoint_callbacks = [c for c in self.trainer.callbacks if isinstance(c, ModelCheckpoint)]
+            if any(c.save_last for c in checkpoint_callbacks):
+                rank_zero_warn('Saving latest checkpoint...')
```

Comment:
just not sure what warn and just info message...

Reply:
It was a warning previously, so I kept it that way. Tell me if you want me to change it

Reply:
I see, I would change it... @awaelchli?

Reply:
I vote for change to info, but no strong feeling, either way is fine for me

Reply:
Done